java-security/src/main/java/com/sap/cloud/security/token/validation/validators/JwtAudienceValidator.java

```diff
@@ -94,6 +93,15 @@ private ValidationResult validateAudienceOfXsuaaBrokerClone(Set<String> allowedA
 				audiences.add(audience);
 			}
 		}
+		// extract audience (app-id) from scopes
+		if (audiences.size() == 0) {
+			for (String scope : token.getClaimAsStringList(TokenClaims.XSUAA.SCOPES)) {
+				if (scope.contains(".")) {
```

Comment:
I would suggest to use the constant DOT here because it is already defined in the class.

Reply:
Done